improvements(README): add more information (#73)
Milerius/nephtys: README.md

| Conventional Commit        | [![Conventional Commits](https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg)](https://conventionalcommits.org)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |

# Nephtys

Nephtys is an open-source **MMORPG** based on ancient egypt written in modern C++.

> This is a `Dofus` like 2.5D, strategic and turn based.

Reviewer: je dirais plutot un truc du genre

Reviewer: ```suggestion
Nephtys is 2.5D isometric game with strategic and turn based oriented battle system.
```

-   [nlohmann::json](https://github.com/nlohmann/json) - Json framework used
-   [EnTT](https://github.com/skypjack/entt) - ECS framework used.
-   [doom::strong_type](https://github.com/doom/strong_type) - C++ strong type library
-   [doom::meta](https://github.com/doom/meta) - C++ metaprogramming utilities
-   [doctest](https://github.com/onqtam/doctest) - Testing framework used

## Code of Conduct

Before any contribution please read our [CODE OF CONDUCT](./CODE-OF-CONDUCT.md).

## Contributing

Please read [CONTRIBUTING.md](./CONTRIBUTING.md), contain the process for submitting pull requests to us.

## Versioning

We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/milerius/nephtys/tags). 

## Authors

-   **Roman Sztergbaum** - _Co-Creator_ - [Milerius](https://github.com/Milerius)
-   **Clément Doumergue** - _Co-Creator_ - [Doom](https://github.com/doom)
-   **Arthur Cros** - _Co-Creator_ - [Riz-cros](https://github.com/Riz-cros)

See also the list of [contributors](./CONTRIBUTORS.md) who participated in this project.

## License

> This project is licensed under the MIT License - see the [LICENSE](./LICENSE) file for details

Reviewer: des fois tu met des point des fois non, faudrais etre coherent

Reviewer: ```suggestion
This project is licensed under the MIT License - see the [LICENSE](./LICENSE) file for details.
```

## Acknowledgments

-   Viktor Kirilov [onqtam](https://github.com/onqtam) for the awesome `doctest` framework.

> -   Michele Caini [skypjack](https://github.com/skypjack) for permanent help and explanations and `EnTT`

Reviewer: des fois tu met des point des fois non, faudrais etre coherent

Reviewer: ```suggestion
-   Michele Caini [skypjack](https://github.com/skypjack) for permanent help and explanations and `EnTT`.
```
